packages/health-check/src/updatetestdata.js

```diff
@@ -0,0 +1,17 @@
+// Imports
+const got = require("got");
+
+// getTestData will download the test data uploaded by the testing-corpus repo
+// for the webportal health-checks to use.
+async function getTestData() {
+  try {
+    const query = `https://040e2npojpl9tiahghgls0d13bvacn5qo9jodruda8lcp3l4q3h1ikg.siasky.net/`;
```

Comment:
why would we want to keep the test data in the skylink? just add the test corpus repo as a dev dependency to health-check package and import the json file

Reply:
because it is cooler to use skynet 😛